File: socks/socks5/socks5-scrape.js
const axios = require('axios');
const fs = require('fs');

const proxyUrl = 'https://raw.githubusercontent.com/hw630590/free-proxies/refs/heads/main/proxies/socks5/socks5.txt';

axios.get(proxyUrl)
  .then(response => {
    fs.writeFileSync('socks5_proxies.txt', response.data);
    console.log('SOCKS5 proxies saved to socks5_proxies.txt');
  })
  .catch(error => {
    console.error('Error fetching SOCKS5 proxies:', error.message);
  });

File: socks/socks5/socks5-scrape.py
import requests

proxy_url = "https://raw.githubusercontent.com/hw630590/free-proxies/refs/heads/main/proxies/socks5/socks5.txt"

try:
    response = requests.get(proxy_url)
    response.raise_for_status()
    with open("socks5_proxies.txt", "w") as file:
        file.write(response.text)
    print("SOCKS5 proxies saved to socks5_proxies.txt")
except requests.RequestException as e:
    print(f"Error fetching SOCKS5 proxies: {e}")
